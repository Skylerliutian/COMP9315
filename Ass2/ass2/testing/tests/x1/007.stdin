N A 10
s A 0
s A 2
s A 4
s A 6
s A 8
O A

N B 10
s B 0
s B 3
s B 4
s B 7
s B 8
O B

o A B

O A