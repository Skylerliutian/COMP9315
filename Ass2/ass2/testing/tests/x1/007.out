0000000101010101
0000000110011001
0000000111011101
